easybuild/framework/easyblock.py

```diff
@@ -1345,10 +1345,12 @@ def make_module_extend_modpath(self):
             # add user-specific module path; use statement will be guarded so no need to create the directories
             user_modpath = build_option('subdir_user_modules')
             if user_modpath:
+                user_envvar = build_option('envvar_user_modules')
                 user_modpath_exts = ActiveMNS().det_user_modpath_extensions(self.cfg)
                 self.log.debug("Including user module path extensions returned by naming scheme: %s", user_modpath_exts)
-                txt += self.module_generator.use(user_modpath_exts, prefix=self.module_generator.getenv_cmd('HOME'),
-                                                 guarded=True, user_modpath=user_modpath)
+                txt += self.module_generator.use(user_modpath_exts,
+                                                 prefix=self.module_generator.getenv_cmd(user_envvar), guarded=True,
```

Comment:
Ideally what I would like to have here is to not have an option at all for this but instead have a generic environment variable that the module checks for and if it doesn't exist then just default it to `$HOME`. In Lmod I can do that with
```
local foo=os.getenv("USER_MODULES") or os.getenv("FOO")
```
but this would require some changes to `self.module_generator.use`

Reply:
However, the nice thing about using environment variables is that I could potentially have a list of them, allowing me to fork out into different paths, e.g., for example a user module built on top of system modules which are built on top of EESSI.

Reply:
From an EESSI user (site) perspective it would maybe make slightly more sense to have SITE_MODULES and USER_MODULES, and HOME. The two first could be a ":" separated list of paths as you say.

Reply:
@akesandgren The way I've implemented it for now, that would mean giving a list on the command line (and order matters) `--envvars-user-modules=USER_MODULES,SITE_MODULES --subdir-user-modules=easybuild`

Reply:
The `EasyBuildMNS` may need an update to support this though, right now it does not fiddle with `MODULEPATH`

Reply:
@damianam Would this work for us at JSC?